src/main/java/com/faforever/api/data/domain/VotingAnswer.java

```diff
@@ -19,7 +17,6 @@
 @Entity
 @Table(name = "voting_answer")
 @Include(name = VotingAnswer.TYPE_NAME, rootLevel = false)
-@ReadPermission(expression = IsEntityOwner.EXPRESSION)
```

Comment:
Why can this be removed?

Reply:
as far as I remember, since this is not root level entity, it is read/accessed via Vote entity which has the read permissions defined, this should be unnecessary.